roles/monit/defaults/main.yml

```diff
@@ -0,0 +1 @@
+alert_email: none@none.com
```

Comment:
may not be necessary since it can be pulled from var/general.yml, but this allows this basic set up to be used if role was pulled out.

Reply:
I also need to answer how this affects variables if I have imported this into playbook. How does the overriding work?

Reply:
http://ansible-manual.readthedocs.io/en/latest/playbooks_variables.html#variables-defined-from-included-files-and-roles

probably be ok.

Reply:
If a user would like to change the monit setup they would need to change the /role/monit/vars/main.yml to overwrite the default.yml